### PR TITLE
Cart management

### PR DESCRIPTION
**Pull request by Pratik1005**
Added following functionality:

- Add product to cart
- Remove product from cart
- Increase / Decrease quantity of product in cart

### CODE REVIEW

**Review by Ash-D23 — APPROVED** [review]
Components looks good , like the way you have divided the cart functionalities

File: src/pages/Cart.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import "../styles/cart.css";
+import {NavMenu, Footer} from "../components/allComponents";
+import {useCart} from "../context/allContext";
+import {CartProductCard} from "../components/allComponents";
+
+const Cart = () => {
+  const {cartState} = useCart();
+  const {cartItems, totalItems, totalPrice} = cartState;
+  let discount = 30 * totalItems;
```
Comment (muskaanshah) [inline src/pages/Cart.js:9]: suggestion- can use const here

**Review by muskaanshah — APPROVED** [review]
Looks good, clean and understandable code

**Review by manismk — APPROVED** [review]
Good naming convention and code is easy to understand

**Pull request merged**